airflow/providers/sftp/operators/sftp.py

```diff
@@ -123,7 +123,7 @@ def execute(self, context: Any) -> str | list[str] | None:
                 f"!= {len(remote_filepath_array)} paths in remote_filepath"
             )
 
-        if not (self.operation.lower() == SFTPOperation.GET or self.operation.lower() == SFTPOperation.PUT):
+        if self.operation.lower() not in (SFTPOperation.GET, SFTPOperation.PUT):
```

Comment:
We discussed and tested a similar change in a different PR, and we found that multiple equality/inequality with and/or is much faster than in/not in tuple, but I think calling `.lower()` once instead of twice might change that.